README.md

```diff
@@ -27,13 +27,6 @@ There is also a test program dialout_server_cli for collecting data from dial-ou
 
 The binaries will be installed under $GOPATH/bin/, they may be copied to any SONiC switch and run there.
 
-You can also build a debian package and install it:
```

Comment:
Have you verified if this should be removed as we discussed? Thanks!

Reply:
I have no idea. I don't know anything about debian. This was something Jipan added. We best ask him directly. @jipanyang

Reply:
Why should it be removed?  Just did a quick verification,  the deb package could be properly generated:
```
jipan@ba32c7f44840:/sonic/tmp$ 
jipan@ba32c7f44840:/sonic/tmp$ git clone https://github.com/Azure/sonic-telemetry.git
Cloning into 'sonic-telemetry'...
remote: Enumerating objects: 501, done.
remote: Total 501 (delta 0), reused 0 (delta 0), pack-reused 501
Receiving objects: 100% (501/501), 1.09 MiB | 0 bytes/s, done.
Resolving deltas: 100% (249/249), done.
jipan@ba32c7f44840:/sonic/tmp$ pushd sonic-telemetry
/sonic/tmp/sonic-telemetry /sonic/tmp
jipan@ba32c7f44840:/sonic/tmp/sonic-telemetry$ dpkg-buildpackage -rfakeroot -b -us -uc
dpkg-buildpackage: info: source package sonic-telemetry
dpkg-buildpackage: info: source version 0.1
dpkg-buildpackage: info: source distribution UNRELEASED

....

dpkg-deb: building package 'sonic-telemetry' in '../sonic-telemetry_0.1_amd64.deb'.
 dpkg-genbuildinfo --build=binary
 dpkg-genchanges --build=binary >../sonic-telemetry_0.1_amd64.changes
dpkg-genchanges: info: binary-only upload (no source code included)
 dpkg-source --after-build sonic-telemetry
dpkg-buildpackage: info: binary-only upload (no source included)
jipan@ba32c7f44840:/sonic/tmp/sonic-telemetry$ popd
/sonic/tmp
jipan@ba32c7f44840:/sonic/tmp$ ls -l
total 5648
drwxr-xr-x 11 jipan gjipan    4096 May 17 00:18 sonic-telemetry
-rw-r--r--  1 jipan gjipan    4902 May 17 00:19 sonic-telemetry_0.1_amd64.buildinfo
-rw-r--r--  1 jipan gjipan    1000 May 17 00:19 sonic-telemetry_0.1_amd64.changes
-rw-r--r--  1 jipan gjipan 5765640 May 17 00:19 sonic-telemetry_0.1_amd64.deb
```

Reply:
Thanks @jipanyang! I think @hui-ma was just interested in trying to remove anything unnecessary and wanted to learn if we should remove this.

Reply:
we should not remove it, it is working.